pkg/util/validation/exporter/exporter.go

```diff
@@ -285,6 +285,12 @@ func (oe *OverridesExporter) Collect(ch chan<- prometheus.Metric) {
 	allLimits := oe.tenantLimits.AllByUserID()
 	for tenant, limits := range allLimits {
 		for _, em := range oe.exportedMetrics {
+			if em.Get(limits) == em.Get(oe.defaultLimits) {
+				// Skip exporting tenant limits that are the same as the default limits.
+				// Note: this comes with an expected tradeoff where metrics, passed via Config.ExtraMetrics, and whose getter
+				// doesn't depend on the passed limit cannot be exposed in the per-tenant limit overrides.
+				continue
+			}
```

Comment:
Note for a reviewer: I think, this should work fine with how GEM augments the set of default limits, via the `validation.MustRegisterExtension`.